File: src/main/java/com/qtech/bubbles/command/BloodMoonCommand.java
package com.qtech.bubbles.command;

import com.qtech.bubbles.common.command.CommandExecutor;
import com.qtech.bubbles.entity.player.PlayerEntity;

public class BloodMoonCommand implements CommandExecutor {
    @Override
    public boolean execute(PlayerEntity player, String[] args) {
        player.getGameType().triggerBloodMoon();
        return true;
    }
}

File: src/main/kotlin/com/qtech/bubbles/command/BloodMoonCommand.kt
package com.qtech.bubbles.command

import com.qtech.bubbles.common.command.CommandExecutor
import com.qtech.bubbles.entity.player.PlayerEntity

open class BloodMoonCommand : CommandExecutor {
    override fun execute(player: PlayerEntity, args: Array<String>): Boolean {
        player.gameMode.triggerBloodMoon()
        return true
    }
}